com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -22,17 +30,15 @@ Specifically, detected Paper white values are often going to produce dimmer resu
 
 #### Neutral
 
-![HDR-Output-Neutral](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-Neutral.png)
-
 | **Property**                         | **Description**                                              |
 | ------------------------------------ | ------------------------------------------------------------ |
-| **Neutral HDR Range Reduction Mode** | The tonemapping curve used for the Neutral tonemapper. The options are:<br />- BT2390: Uses a curve defined by the BT2390 broadcasting recommendations. (Default)<br />- Reinhard: A very simple tonemapping operator.<br /><br />This option is available only when Additional Properties are displayed. |
+| **Neutral HDR Range Reduction Mode** | The curve that the Player uses for tone mapping. The options are:<br />- BT2390: The default. Defined by the [BT.2390](https://www.itu.int/pub/R-REP-BT.2390) broadcasting recommendations.<br />- Reinhard: A very simple tonemapping operator.<br /><br />This option is available only when Additional Properties are displayed. |
```

Comment:
Need to add the link to Additional Properties as indicated https://github.com/Unity-Technologies/Graphics/pull/5992#discussion_r744688164

Reply:
@FrancescoC-unity